open_discussions/settings.py

```diff
@@ -669,6 +669,7 @@
 ELASTICSEARCH_MIN_QUERY_SIZE = get_int("ELASTICSEARCH_MIN_QUERY_SIZE", 2)
 ELASTICSEARCH_MAX_SUGGEST_HITS = get_int("ELASTICSEARCH_MAX_SUGGEST_HITS", 1)
 ELASTICSEARCH_MAX_SUGGEST_RESULTS = get_int("ELASTICSEARCH_MAX_SUGGEST_RESULTS", 1)
+ELASTICSEARCH_SHARD_COUNT = get_int("ELASTICSEARCH_SHARD_COUNT", 5)
```

Comment:
I found a default of 5 to fail locally when reindexing, if a default of 5 makes sense for deployment scenarios, could you add `ELASTICSEARCH_SHARD_COUNT: 1` to the `docker-compose` environment variables so local development works out of the box? 

https://github.com/mitodl/open-discussions/blob/master/docker-compose.yml#L3

Reply:
5 shards was the default in Elasticsearch for a long time until version 6 or 7, and my recollection is that an earlier version is being used here. I'm happy to set the default lower, though. There's no reason for it to be higher than 1 or 2 in a development context.

Reply:
either way is fine by me, locally for whatever reason this new setting causes ES to hard throttle the reindexing so it ends up failing. that probably needs to be fixed too...

Reply:
I've just changed the default to 2. Does it work better for you?